examples/kitty/Makefile

```diff
@@ -204,7 +224,7 @@ NETWORK_COMPONENT_CFLAGS := \
 	-MD \
 	-MP
 
-NETWORK_COMPONENTS_LDFLAGS := -L$(BOARD_DIR)/lib -L$(SDDF)/lib -L$(LIBGCC)
+NETWORK_COMPONENTS_LDFLAGS := -L$(BOARD_DIR)/lib -L$(LIBGCC) -L$(LIBC)
```

Comment:
The network components do not need libgcc, only NFS. I know it's not from this PR but could you try remove it and see what happens?

Reply:
That seems to work, everything compiles, nfs works. Will push these changes.